.github/workflows/compile-artifact.yml

```diff
@@ -8,6 +8,12 @@ on:
         default: "2.6.8"
         required: false
         type: "string"
+      gemfile_suffix:
+        description: "The suffix to add to the Gemfile (for multi-gem projects)"
```

Comment:
worth adding that it needs to include the . at the start

Reply:
Definitely